packages/rs-dpp/src/data_contract/document_type/class_methods/try_from_schema/v0/mod.rs

```diff
@@ -549,23 +593,16 @@ fn insert_values_nested(
     property_key: String,
     property_value: &Value,
     root_schema: &Value,
-) -> Result<(), ProtocolError> {
+) -> Result<(), DataContractError> {
```

Comment:
Shouldn't we do ProtocolError::DataContract(DataContractError::... )?

Reply:
No, DataContractError can either be a basic consensus error, or a protocol error, depending on how you call this method.

Reply:
I see. Right now we call them without map_err